slushfile.js

```diff
@@ -1,23 +1,29 @@
-const gulp = require('gulp'),
-	conflict = require('gulp-conflict'),
-	template = require('gulp-template'),
-	path = require('path'),
-	inquirer = require('inquirer'),
-	_ = require('lodash'),
-	fs = require('fs'),
-	execS = require('child_process').execSync;
+const gulp = require('gulp');
+const conflict = require('gulp-conflict');
+const template = require('gulp-template');
+const path = require('path');
+const inquirer = require('inquirer');
+const _ = require('lodash');
+const fs = require('fs');
+const execS = require('child_process').execSync;
 
-const defaults = {
-	moduleNaturalName: process.argv[3] || 'htz-',
+const GetSafeName = {
```

Comment:
OK if you wish it so, but I would argue it would be more readable as a class:
````
class ModuleOptions{
moduleNaturalName: process.argv[3] || 'htz-';
moduleDescription: '';
moduleAuthorName: execS('git config user.name', { encoding: 'utf8' }).split('\n')[0];
moduleAuthorEmail: execS('git config user.email', { encoding: 'utf8' }).split('\n')[0];

get moduleSafeName() {
  		return this.moduleNaturalName.replace(' ', '-');
  	}
}
````

obviously you need to instantiate with 'new';
but if you prefer it your way - OK to merge